doc/installation.md

```diff
@@ -78,15 +78,16 @@ if not the SDK will use a timer polling approach which is less sensitive for dev
    Notice: You can always remove permissions by running: `./scripts/setup_udev_rules.sh --uninstall`
 
 3. Build and apply patched kernel modules for:
-    * Ubuntu 20/22 (focal/jammy) with LTS kernel 5.13, 5.15, 5.19, 6.2, 6.5 \
+    * Ubuntu 20/22/24 (focal/jammy/noble) with LTS HWE kernel 5.15, 5.19, 6.5 \
```

Comment:
Is the `HWE` really needed? Up to you, I found it confusing...

Reply:
Done